osfoffline/polling_osf_manager/polling.py

```diff
@@ -76,9 +76,17 @@ def handle_exception(self, future):
         self._loop.call_later(5, self.start)
 
     def start(self):
+        try:
```

Comment:
All polyfills should be placed at the top of the module.

Our codebase should be written as though it were using the latest version of the code. All special cases should be in one clear and obvious place.